blackbox/docs/admin/monitoring.rst

```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
 
    Defines if the node is able to process SQL statements.
 
+Exposing JMX via HTTP
+=====================
+
+The JMX metrics and also a readiness endpoint can be exposed via HTTP by using
+the `Crate JMX HTTP Exporter`_ java agent. See the `README`_ of the `Crate JMX
+HTTP Exporter`_ repository for further information and usage details.
```

Comment:
I would keep this short(er) by stopping after "information". ( "and usage details" doesn't add much value IMO)